### PR TITLE
Clear screen works irregardless of the platform

### PR DESCRIPTION
**Pull request by shaw-rohit**
This PR addresses issue #23. Tested on Windows 10 and Ubuntu 19.10.

### CODE REVIEW

**Comment by arumoy-shome** [comment]
@shaw-rohit can you please remove the __pycache__/ dirs, I have opened #27 which should ignore them but until then I would be careful when using `git add .` 😄

**Comment by shaw-rohit** [comment]
> @shaw-rohit can you please remove the **pycache**/ dirs, I have opened #27 which should ignore them but until then I would be careful when using `git add .` 😄

Done!

**Comment by shrutirao94** [comment]
Works for me now! 👍

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Byte-compiled
 .cache
 .pytest_cache/
-__pycache__/
+**/*/__pycache__/
```
Comment (arumoy-shome) [inline .gitignore:4]: 🔥

**Comment by shaw-rohit** [comment]
> Don't think we need this PR seeing that #29 already implements this?

#29 is based on this branch, hence the reason why they look similar. Merging this before #29 would be ideal.

**Comment by arumoy-shome** [comment]
In that case, feel free to 🐑

**Pull request merged**